frontend/pages/index.tsx

```diff
@@ -93,12 +94,27 @@ const TopSection = ({
 }) => {
   const { t } = useTranslation()
 
-  const [selectedName, setSelectedName] = useState(topApps[0].name)
+  const [selectedName, setSelectedName] = useState("")
+  const [selectedIndex, setSelectedIndex] = useLocalStorage(
+    "selected-category",
+    0,
```

Comment:
I would prefer to save the name not the index

Reply:
Changed local storage to store name instead of index.